config/sites/admissions.uiowa.edu/views.view.majors.yml

```diff
@@ -386,6 +387,464 @@ display:
+          disable_sql_rewrite: false
+          distinct: false
+          replica: false
+          query_tags: {  }
+      defaults:
+        query: false
+        title: false
+        group_by: false
+        relationships: false
+        fields: false
+        sorts: false
+        arguments: false
+        filters: false
+        filter_groups: false
+        header: false
+        footer: false
+      relationships:
+        field_major_community_college:
+          id: field_major_community_college
+          table: node__field_major_community_college
+          field: field_major_community_college
+          relationship: none
+          group_type: group
+          admin_label: 'field_major_community_college: Taxonomy term'
+          plugin_id: standard
+          required: true
+      group_by: true
+      display_description: 'Page listing all community colleges associated with at least one 2 plus 2 major.'
+      header:
+        area:
+          id: area
+          table: views
+          field: area
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: text
+          empty: false
+          content:
+            value: '<p>All Iowa community colleges participate in the 2 Plus 2 plan. </p>'
+            format: filtered_html
+          tokenize: false
+      footer:
+        area:
+          id: area
+          table: views
+          field: area
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: text
+          empty: false
+          content:
+            value: "<h2>Available majors</h2>\r\n<p>We are continually expanding the number of majors available at Iowa under the 2 Plus 2 plan. <a href=\"/2plus2/majors\">See the 2 Plus 2 areas of study</a> available at the University of Iowa.</p>"
+            format: filtered_html
+          tokenize: false
+      display_extenders:
+        metatag_display_extender:
+          metatags: {  }
+          tokenize: false
+      path: 2plus2/colleges
+    cache_metadata:
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - 'user.node_grants:view'
+        - user.permissions
+      tags: {  }
+  available_majors:
+    id: available_majors
+    display_title: 'Available Majors'
+    display_plugin: page
+    position: 3
+    display_options:
+      title: 'Available Majors for 2 Plus 2 Students'
+      fields:
+        title:
+          id: title
+          table: node_field_data
+          field: title
+          relationship: none
+          group_type: group
+          admin_label: ''
+          entity_type: node
+          entity_field: title
+          plugin_id: field
+          label: ''
+          exclude: false
+          alter:
+            alter_text: false
+            text: '<a href="/2plus2/majors/{{ title|lower|replace({''/'':''''})|replace({'' '':''-''}) }}">{{ title }}</a>'
```

Comment:
@cory-skeers should we remove this code now that it is being done via preprocess?

Reply:
Forgot that unchecking the overwrite doesn't remove that--taken out.